src/Microsoft.DocAsCode.Dfm.VscPreview/DfmServiceProvider.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DfmProcess
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+
+
+    [Export("dfm", typeof(IMarkdownServiceProvider))]
```

Comment:
>dfm [](start = 13, length = 3)

duplicate dfm service provider name with cuurently 'dfm', could we reuse existing DfmServiceProvider? just passing null templateDir as parameter